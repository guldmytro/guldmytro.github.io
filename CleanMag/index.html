<!DOCTYPE html>
<html>
<head>
	<meta charset="utf-8">
	<title>CleanMag</title>
	<!--[if lt IE 9]> 
 	<script src="http://html5shiv.googlecode.com/svn/trunk/html5.js"></script> 
	<![endif]-->
	<link rel="stylesheet" type="text/css" href="normalize.css">
	<link rel="stylesheet" type="text/css" href="style.css">
</head>
<body>
	<section id="logo" class="clearfix">
			<figure>Clean <span class="blue">Mag</span></figure>
			<div class="banner">728x90 Banner</div>
	</section>
	<nav id="top">
		<div class="container clearfix">
			<ul>
				<li>Home</li>
				<li>About Me</li>
				<li>Design</li>
				<li>Services</li>
				<li>Request</li>
				<li>Quot</li>
			</ul>
		</div>
	</nav>
	<nav id="bottom">
		<div class="container">
			<ul>
				<li class="hovered">insipration</li>
				<li>technology</li>
				<li>freebies</li>
				<li>wordpress</li>
				<li>tutorials</li>
				<li>fonts</li>
				<li>wallpapers</li>
			</ul>
		</div>
	</nav>
	<div id="wrapper">
		<div class="container clearfix">
			<div class="wr-left">
				<div class="big-picture">
					<div class="arrow">
						<h3>featured</h3>
					</div>
					<div class="dark-section"></div>
					<div class="lists clearfix">
							<div class="list">1</div>
							<div class="list">2</div>
							<div class="list">3</div>
					</div>
				</div>
				<article class="left-article">
					<div class="picture"></div>
					<div class="text">
						<div class="clearfix">
						<h5><span>Admin</span> - Nov 28th, 2010</h5>
						<h5><span>10 Comments</span></h5>
						</div>
						<h3>Thanksgiving greeting card PSD</h3>
						<p>Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Aenean commodo ligula eget dolor. Aenean massa. Cum sociis natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus.</p>
						<h5><span>Read more</span></h5>
					</div>
				</article>
				<article class="left-article">
					<div class="picture"></div>
					<div class="text">
						<div class="clearfix">
						<h5><span>Admin</span> - Nov 28th, 2010</h5>
						<h5><span>10 Comments</span></h5>
						</div>
						<h3>Vintage business card PSD (front & back)</h3>
						<p>Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Aenean commodo ligula eget dolor. Aenean massa. Cum sociis natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus.</p>
						<h5><span>Read more</span></h5>
					</div>
				</article>
				<article class="left-article">
					<div class="picture"></div>
					<div class="text">
						<div class="clearfix">
						<h5><span>Admin</span> - Nov 28th, 2010</h5>
						<h5><span>10 Comments</span></h5>
						</div>
						<h3>Request a quote form PSD in 3 colors</h3>
						<p>Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Aenean commodo ligula eget dolor. Aenean massa. Cum sociis natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus.</p>
						<h5><span>Read more</span></h5>
					</div>
				</article>
				<article class="left-article">
					<div class="picture"></div>
					<div class="text">
						<div class="clearfix">
						<h5><span>Admin</span> - Nov 28th, 2010</h5>
						<h5><span>10 Comments</span></h5>
						</div>
						<h3>PSD book template & icons</h3>
						<p>Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Aenean commodo ligula eget dolor. Aenean massa. Cum sociis natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus.</p>
						<h5><span>Read more</span></h5>
					</div>
				</article>
				<h2>1  2  3  4  of  75  <span>Next</span></h2>
				<div class="silver-line"></div>
				<div class="content clearfix">
					<div class="news">
						<h3>user news</h3>
						<h6>Thanksgiving greeting card PSD</h6>
						<h5>November 29, 2010 by <span>Admin</span></h5>
						<p>Maecenas tempus, tellus eget condimentum rhoncus, sem quam semper libe</p>
						
						<h6>Maecenas nec odio et ante</h6>
						<h5>November 29, 2010 by <span>Admin</span></h5>
						<p>Maecenas tempus, tellus eget condimentum rhoncus, sem quam semper libe</p>
						
						<h6>Cras dapibus</h6>
						<h5>November 29, 2010 by <span>Admin</span></h5>
						<p>Maecenas tempus, tellus eget condimentum rhoncus, sem quam semper libe</p>

						<h6>Sed fringilla</h6>
						<h5>November 29, 2010 by <span>Admin</span></h5>
						<p>Maecenas tempus, tellus eget condimentum rhoncus, sem quam semper libe</p>
						<h5 class="link clearfix"><span>Read more news</span></h5>
						<div class="clear"></div>
						<button>+ Submit News</button>
					</div>
					<div class="news">
						<h3>random inspiration</h3>
						<div class="pic-container clearfix">
							<div class="pics"></div>
							<div class="pics"></div>
							<div class="pics"></div>
							<div class="pics"></div>
							<div class="pics"></div>
							<div class="pics"></div>
						</div>
						<h3>flickr group</h3>
						<div class="pic-container clearfix">
							<div class="pics"></div>
							<div class="pics"></div>
							<div class="pics"></div>
							<div class="pics"></div>
							<div class="pics"></div>
							<div class="pics"></div>
						</div>
					</div>
				</div>
			</div>
			<div class="wr-right">
				<form class="clearfix">
					<input type="text" name="search" placeholder="Search Blog" />
					<div id="banner_shit" class="clearfix">
						<div class="one"><h1>1234</h1>
							<h3>Readers</h3>
							<h5 class="link clearfix"><span>Subscribe now</span></h5>
						</div>
						<div class="two"><h1>1200</h1>
							<h3>Twitter Followers</h3>
							<h5 class="link clearfix"><span>Follow on Twitter</span></h5>
						</div>
					</div>
					<input type="email" name="email" placeholder="Enter email address" />
					<button>subscribe</button>
				</form>
				<figure class="clearfix">
					<div class="picz"></div>
					<div class="picz"></div>
					<div class="picz"></div>
					<div class="picz"></div>
				</figure>
				<h3>categories</h3>
				<nav class="clearfix">
					<div class="clearfix">
						<h4>Backgrounds</h4>
						<h4>Brushes</h4>
						<h4>Design Articles</h4>
						<h4>Fonts</h4>
						<h4>Free PSD Files</h4>
						<h4>Icons</h4>
						<h4>Logos</h4>
						<h4>Patterns</h4>
						<h4>Textures</h4>
						<h4>Tools & Utilities</h4>
					</div>
					<div class="clearfix">
						<h4>Backgrounds</h4>
						<h4>Brushes</h4>
						<h4>Design Articles</h4>
						<h4>Fonts</h4>
						<h4>Free PSD Files</h4>
						<h4>Icons</h4>
						<h4>Logos</h4>
						<h4>Patterns</h4>
						<h4>Textures</h4>
						<h4>Tools & Utilities</h4>
					</div>
				</nav>
				<h3>latest posts</h3>
				<div class="posts clearfix">
					<div class="l-posts">
						<div class="rect"></div>
						<div class="rect"></div>
						<div class="rect"></div>
						<div class="rect"></div>
						<div class="rect"></div>
						<div class="rect"></div>
					</div>
					<div class="r-posts">
						<div class="trect">Request a quote form PSD in 3 colors</div>
						<div class="trect">Quot ubique mediocritatem</div>
						<div class="trect">Modus primis</div>
						<div class="trect">Vidit omnis labore</div>
						<div class="trect">Mei iisque reformidans ei</div>
						<div class="trect">Iusto dicit tincidunt per eu</div>
					</div>
				</div>
				<div class="fb-banner">
					<div class="inside-fb">
						<header><img src="img/fb.png" width="85px" height="17px"></header>
						<div class="with-like"><h5 id="face">GraphicsFuel</h5><button>Like</button></div>
						<h5 class="faced">288 people like GraphicsFuel</h5>
						<div class="social-pics">
							<div class="s-pic">Eric</div>
							<div class="s-pic">Kelley</div>
							<div class="s-pic">Jitendra Kan</div>
							<div class="s-pic">Galax</div>
							<div class="s-pic">German</div>
						</div>
					</div>
				</div>
				<div class="another-banner">300x120 Banner</div>
				<h3>Videos</h3>
				<video controls poster="video/poster.jpg">
					<source src="video/SAIL - AWOLNATION (Unofficial Video).mp4" type="video/mp4">
					Your browser does not support the video tag.
				</video>

			</div>
		</div>
	</div>
	<footer>
		<div class="container clearfix">
			<div class="block">
				<h3>About</h3>
				<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Proin Lorem ipsum dolor sit amet, consectetur adipiscing elit. Proin Lorem ipsum dolor sit amet, consectetur adipiscing elit. Proin Lorem ipsum dolor sit amet, consectetur adipiscing elit. Proin Lorem ipsum dolor sit amet, consectetur adipiscing elit. Proin </p>
			</div>
			<div class="block clearfix">
				<h3>browse</h3>
				<div class="left">
					<h4>Insipration</h4>
					<h4>Technology</h4>
					<h4>Freebies</h4>
					<h4>Wordpress</h4>
					<h4>Tutorials</h4>
					<h4>Fonts</h4>
					<h4>Wallpapars</h4>
				</div>
				<div class="left">
					<h4>Home</h4>
					<h4>About Me</h4>
					<h4>Design Services</h4>
					<h4>Request Quote</h4>
					<h4>Advertise</h4>
					<h4>Contact Me</h4>
				</div>

			</div>
			<div class="block">
				<h3>write for us</h3>
				<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Proin Lorem ipsum dolor sit amet, consectetur adipiscing elit. Proin Lorem ipsum dolor sit amet, consectetur adipiscing elit. Proin Lorem ipsum dolor sit am </p>
			</div>
			
		</div>
	</footer>
	
</body>
</html>