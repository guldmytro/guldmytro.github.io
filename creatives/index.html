<!DOCTYPE html>
<html>
<head>
	<title>Creatives</title>
	<link rel="stylesheet" type="text/css" href="normalize.css">
	<link rel="stylesheet" type="text/css" href="style.css">
</head>
<body>
<div id="page-wrap">
	<header>
		<div class="container clearfix">
			<figure class="logo left">CS</figure>
			<figure class="shadow left"></figure>
			<ul class="menu right clearfix">
				<li class="activate"><a href="">Home</a></li>
				<li><a href="">Pages</a></li>
				<li><a href="">Features</a></li>
				<li><a href="">Extensions</a></li>
				<li><a href="">Tutorials</a></li>
				<li><a href="">Contact us</a></li>
				<li><a href=""><img src="img/facebook.svg"></a></li>
				<li><a href=""><img src="img/twitter.svg"></a></li>
				<li><a href=""><img src="img/skype.svg"></a></li>
				<li><a href=""><img src="img/instagram.svg"></a></li>
			</ul>
		</div>
		<hr>
		<div class="container">
			<h1>creatives</h1>
			<span>Power by PSDfreebies.com</span>
			<div class="scroll">
				Scroll Down
				<br>
				<img src="img/down-arrow.svg" width="67px" height="67px">
			</div>
		</div>
		<div id="back">W</div>
		<div id="Awesome">We Are An Awesome Agency</div>
		<div class="blue"></div>
	</header>
	<section id="main-menu">
		<div class="container clearfix">
			<div class="left"><figure class="circle"><img src="img/Write.png"></figure><h3>future one</h3><p>Aenean lacinia bibendum nulla sed consectetur. Vivamus sagittis lacus vel augue laoreet rutrum faucibus</p></div>
			<div class="left"><figure class="circle"><img src="img/copy.svg"></figure><h3>future two</h3><p>Aenean lacinia bibendum nulla sed consectetur. Vivamus sagittis lacus vel augue laoreet rutrum faucibus</p></div>
			<div class="left"><figure class="circle"><img src="img/Search.png"></figure><h3>future three</h3><p>Aenean lacinia bibendum nulla sed consectetur. Vivamus sagittis lacus vel augue laoreet rutrum faucibus</p></div>
			<div class="left"><figure class="circle"><img src="img/Simon.png"></figure><h3>future four</h3><p>Aenean lacinia bibendum nulla sed consectetur. Vivamus sagittis lacus vel augue laoreet rutrum faucibus</p></div>
		</div>
	</section>
	<section class="photo clearfix">
		<div class="block right"></div>	
		<div class="block right"><div class="half-container right"><time>Jan 4, 2016 | In Culture</time> <h2>Akita Inu Dog</h2>
			<p>Cum sociis natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec sed odio dui. Vivamus sagittis lacus vel augue laoreet rutrum faucibus dolor auctor.</p>
			<p>Donec ullamcorper nulla non metus auctor fringilla. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed posuere consectetur </p>
		</div></div>
	</section>
	<section class="photo clearfix">
		<div class="block left"></div>	
		<div class="block left"><div class="half-container left"><time>Jan 4, 2016 | In Culture</time> <h2>Akita Inu Dog</h2>
			<p>Cum sociis natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec sed odio dui. Vivamus sagittis lacus vel augue laoreet rutrum faucibus dolor auctor.</p>
			<p>Donec ullamcorper nulla non metus auctor fringilla. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed posuere consectetur </p></div></div>	
	</section>
	<section id="blog">
		<div class="container">
			<h4>Let's blog</h4>
			<span>B</span>
			<p>We love clean design and advanced digital solutions.</p>
			<div class="pictures clearfix">
				<div class="ar left"><img src="img/pic1.png"><h3>Fresh Summer Campaign
				</h3><p>Cum sociis natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec sed odio dui. Vivamus sagittis lacus vel augue</p></div>
				<div class="ar left"><img src="img/pic2.png"><h3>Fresh Summer Campaign
				</h3><p>Cum sociis natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec sed odio dui. Vivamus sagittis lacus vel augue</p></div>
				<div class="ar left"><img src="img/pic3.png"><h3>Fresh Summer Campaign
				</h3><p>Cum sociis natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec sed odio dui. Vivamus sagittis lacus vel augue</p></div>
				<div class="ar left"><img src="img/pic4.png"><h3>Fresh Summer Campaign
				</h3><p>Cum sociis natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec sed odio dui. Vivamus sagittis lacus vel augue</p></div>
			</div>
		</div>
	</section>
	<section id="first-footer">
		<div class="container clearfix">
			<div class="block-footer left">Cum sociis natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec sed odio dui. Vivamus sagittis lacus vel augue laoreet rutrum faucibus </div>
			<div class="block-footer left">Cum sociis natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec sed odio dui. Vivamus sagittis lacus vel augue laoreet rutrum faucibus </div>
		</div>
	</section>
	<section id="about">
		<div class="container clearfix">
			<article class="left"><h2>Akita Inu Dog</h2>
				<p>Cum sociis natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec sed odio dui. Vivamus sagittis lacus vel augue laoreet rutrum faucibus dolor auctor.</p>
				<div class="btn">Fully Responsive Design</div>
				<div class="btn">High Quality Code</div>
				<div class="btn">Different Headers & Layout</div>
			</article>
			<article class="left"></article>
		</div>
	</section>
	<section id="team">
		<div class="container clearfix">
			<h4>our team</h4>
			<span>t</span>
			<p>sales long tail influencer pitch release niche market.</p>
			<div class="player left"><div class="p-name"><h5>JOHN SNOW</h5><h6>CEO</h6></div></div>		
			<div class="player left"><div class="p-name"><h5>LADY SANSA</h5><h6>WebDesigner</h6></div></div>	
			<div class="player left"><div class="p-name"><h5>DAVID SMITH</h5><h6>Photographer</h6></div></div>
			<div class="player left"><div class="p-name"><h5>LADY SANSA</h5><h6>Photographer</h6></div></div>
		</div>
	</section>
	<section id="connect">
		<div class="container">
			<h4> We’d love to hear about your project</h4>
			<span>C</span>
			<form action="#" class="clearfix">
				<input placeholder="Name" id="name" class="left" type="text">
				<input placeholder="Email" id="email" class="left" type="email">
				<input placeholder="Phone" id="name" class="left" type="text">
				<textarea placeholder="Message" id="yourMessage" class="left" cols="0" rows="0"></textarea>
			</form>
			<div class="bt">submit</div>
		</div>
	</section>
	<section id="map"></section>
	<footer>
		<div class="container">
			<div class="left">&copy; 2016 PSDfreebies.com – All Right Reserved</div>
			<div class="right">Design by:<img src="img/bee.svg" width="49.6px" height="40"> PSDFreebies.com</div>
		</div>
	</footer>
</div>
</body>
</html>