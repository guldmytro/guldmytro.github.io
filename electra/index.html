<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Elektra</title>
    <link rel="stylesheet" href="./css/main.css">
</head>
<body>
    <header class="header">
        <div class="container container_fluid">
            <div class="header-row">
                <div class="logo">
                    <img src="./img/logo.png" width="404" height="152" alt="flash and elektra" class="logo__img">
                    <img src="./img/logo.png" width="404" height="152" alt="flash and elektra" class="logo__img_hovered">
                </div>
                <button type="button" class="header__btn">
                    <span class="hovered">Coming Soon</span>
                    <span class="static">Launch app</span>
                </button>
            </div>
        </div>
    </header>
    <section class="firstscreen">
        <div class="spot"></div>
        <div class="spot"></div>
        <div class="spot"></div>
        <div class="container">
            <div class="firstscreen-row">
                <h1 class="firstscreen__title">We make <span class="gradient">NFT</span> bridging simple</h1>
            </div>
        </div>
    </section>
    <main class="main">
        <div class="container">
            <div class="main-sections">
                <section class="main-sections__item section-item">
                    <div class="section-item__text">
                        <h2 class="section-item__header"><span class="gradient">About Elektra</span></h2>
                        <p class="section-item__desc">Elektra is a fully composable NFT cross chain bridging protocol which enables the seamless transfer of value across different blockchains.</p>
                        <p class="section-item__desc_darken">We aim to create an NFT primitive that removes the tribal chain dependancy currently seen in the market and work towards building an omnichain future.</p>
                    </div>
                </section>
                <section class="main-sections__item section-item section-item_picture">
                    <div class="section-item__text">
                        <h2 class="section-item__header section-item__header_bordered"><span>Use Cases</span></h2>
                        <p class="section-item__desc smaller">Elektra will enable users to take advantage of liquidity wherever it is and enable the transition of NFTs to be used in DeFi based applications such as lending markets and DEXs.</p>
                        <p class="section-item__desc_darken smaller">We believe the biggest use cases of our bridge is bringing interoperability in the following categories: gaming, lending, cross chain trading and general transfer of value.</p>
                    </div>
                    <div class="section-item__picture">
                        <img src="./img/planet.png" width="869" height="611" alt="planet" loading="lazy">
                        <div class="spot"></div>
                    </div>
                </section>
                <section class="main-sections__item section-item section-item_picture invert">
                    <div class="section-item__text">
                        <h2 class="section-item__header section-item__header_bordered">
                            <span>DAO</span>
                        </h2>
                        <p class="section-item__desc smaller">Elektra aims to become a DAO governed by the community, EKA will play a central role in voting, incentivising early users and bootstrapping the network.</p>
                        <p class="section-item__desc_darken smaller">Stakers of EKA receive 100% of the protocol revenue. EKA liquidity providers will be able to lock their LP tokens to earn rewards.</p>
                    </div>
                    <div class="section-item__picture">
                        <img src="./img/shield.png" width="657" height="841" alt="shield" loading="lazy">
                    </div>
                </section>
                <section class="main-sections__item section-item section-item_picture">
                    <div class="section-item__text">
                        <h2 class="section-item__header"><span class="gradient">Come for the memes, stay for the protocol</span></h2>
                        <p class="section-item__desc">While NFT bridging is fun, follow us on Twitter for funny memes and alpha from the community</p>
                        <div class="spot"></div>
                    </div>
                    <div class="section-item__picture">
                        <img src="./img/people.png" width="1368" height="987" alt="people" loading="lazy">
                    </div>
                </section>
            </div>
        </div>
    </main>
    <footer class="footer">
        <div class="container container_fluid">
            <div class="footer-row">
                <div class="logo">
                    <img src="./img/logo.png" alt="flash and elektra" class="logo__img">
                    <img src="./img/logo.png" alt="flash and elektra" class="logo__img_hovered">
                </div>
                <p class="footer__copy">â“’ 2022 Elektra. All rights reserved</p>
                <ul class="footer__socials footer-socials">
                    <li class="footer-socials__item socials-item">
                        <a href="#" class="socials-item__link" target="_blank" aria-label="our twitter">
                            <img src="./img/ic-twitter.svg" width="31" height="25" alt="twitter">
                        </a>
                    </li>
                    <li class="footer-socials__item socials-item">
                        <a href="#" class="socials-item__link" target="_blank" aria-label="our telegram">
                            <img src="./img/ic-telegram.svg" width="27" height="23" alt="telegram">
                        </a>
                    </li>
                </ul>
            </div>
        </div>
    </footer>
</body>
</html>
