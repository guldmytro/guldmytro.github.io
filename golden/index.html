<!DOCTYPE html>
<html>
<head>
	<meta charset="UTF-8">
	<title>Golden</title>
	<link rel="stylesheet" href="normalize.css" type="text/css">
	<link rel="stylesheet" href="style.css" type="text/css">
</head>
<body>
	<div id="wrapper">
		<div id="header">
			<div id="container" class="clearfix">
				<div class="logo">
					<a href="#">
						<img src="img/Logo-Treehouse.png" alt="logo">
					</a>
				</div>
				<ul class="nav">
					<li><a href="#">Home</a></li>
					<li><a href="#">Services</a></li>
					<li><a href="#">Portfolio</a></li>
					<li><a href="#">About</a></li>
					<li><a href="#">Contact</a></li>
				</ul>
			</div>
		</div>

		<div id="home">
			<div class="container">
				<h3>Welcome To Our Studio!</h3>
				<h1>Itâ€™s nice to Meet you</h1>
				<button class="btn">Tell Me More</button>
			</div>
		</div>

		<div id="services" class="box">
			<div class="container">
				<h2 class="title">Services</h2>
				<p class="descr">Proin iaculis purus consequat sem cure.</p>

				<div class="services-wrap clearfix">
					<div class="services-item commerce">
						<h5 class="title">E-Commerce</h5>
						<p>Proin iaculis purus consequat sem cure 
							digni ssim. Donec porttitora entum suscipit
						</p>
					</div>
					<div class="services-item responsive">
						<h5 class="title">Responsive Web</h5>
						<p>Proin iaculis purus consequat sem cure 
							digni ssim. Donec porttitora entum suscipit
						</p>
					</div>
					<div class="services-item security">
						<h5 class="title">Web Security</h5>
						<p>Proin iaculis purus consequat sem cure 
							digni ssim. Donec porttitora entum suscipit
						</p>
					</div>
				</div>
			</div>
		</div>

		<div id="portfolio" class="box">
			<div class="container">
				<h2 class="title">Our portfolio</h2>
				<p class="descr">Proin iaculis purus consequat sem cure.</p>
				
				<div class="portfolio-wrap clearfix">
					<div class="portfolio-item">
						<a href=""><img src="img/p-item1.jpg" alt="Ebony & Ivory"></a>
						<div class="p-item-footer">
							<h5 class="title">Ebony & Ivory</h5>
							<span>Branding</span>
						</div>
					</div>
					<div class="portfolio-item">
						<a href=""><img src="img/p-item2.jpg" alt="Ebony & Ivory"></a>
						<div class="p-item-footer">
							<h5 class="title">Smart Stationary</h5>
							<span>Pvrint Design</span>
						</div>
					</div>
					<div class="portfolio-item">
						<a href=""><img src="img/p-item3.jpg" alt="Ebony & Ivory"></a>
						<div class="p-item-footer">
							<h5 class="title">Clever Poster</h5>
							<span>Print Design</span>
						</div>
					</div>
					<div class="portfolio-item">
						<a href=""><img src="img/p-item4.jpg" alt="Ebony & Ivory"></a>
						<div class="p-item-footer">
							<h5 class="title">Vinyl Record</h5>
							<span>Product Mock-Up</span>
						</div>
					</div>
					<div class="portfolio-item">
						<a href=""><img src="img/p-item5.jpg" alt="Ebony & Ivory"></a>
						<div class="p-item-footer">
							<h5 class="title">Treehouse Template</h5>
							<span>Web Design</span>
						</div>
					</div>
					<div class="portfolio-item">
						<a href=""><img src="img/p-item6.jpg" alt="Ebony & Ivory"></a>
						<div class="p-item-footer">
							<h5 class="title">Burned Logo</h5>
							<span>Branding</span>
						</div>
					</div>
				</div>
			</div>
		</div>

		<div id="about" name="about" class="box">
			<div class="container">
				<h2 class="title">About us</h2>
				<p class="descr">Proin iaculis purus consequat sem cure.</p>
				<div class="about-wrap">
					<div class="about-item clearfix">
						<div class="about-text left text-right">
							<h5 class="title"><span>July 2010</span><br />Our Humble Beginnings</h5>
							<p>Proin iaculis purus consequat sem cure 
								digni ssim. Donec porttitora entum suscipit 
								aenean rhoncus posuere odio in tincidunt. Proin 
								iaculis purus consequat sem cure digni 
								ssim. Donec porttitora entum suscipit.
							</p>
						</div>
						<div class="about-img left">
							<img src="img/Image.png" />
						</div>
					</div>

					<div class="about-item clearfix">
						<div class="about-text right text-left">
							<h5 class="title"><span>January 2011</span><br />Facing Startup Battles</h5>
							<p>Proin iaculis purus consequat sem cure 
								digni ssim. Donec porttitora entum suscipit 
								aenean rhoncus posuere odio in tincidunt. Proin 
								iaculis purus consequat sem cure digni 
								ssim. Donec porttitora entum suscipit aenean 
								rhoncus posuere odio in tincidunt.
							</p>
						</div>
						<div class="about-img right">
							<img src="img/Image1.png" />
						</div>
					</div>

					<div class="about-item clearfix">
						<div class="about-text left text-right">
							<h5 class="title"><span>December 2012</span><br />Enter The Dark Days</h5>
							<p>Proin iaculis purus consequat sem cure 
								digni ssim. Donec porttitora entum suscipit 
								aenean rhoncus posuere odio in tincidunt. Proin 
								iaculis purus consequat sem cure digni.
							</p>
						</div>
						<div class="about-img left">
							<img src="img/Image2.png" />
						</div>
					</div>

					<div class="about-item clearfix">
						<div class="about-text right text-left">
							<h5 class="title"><span>february 2014</span><br />Our Triumph</h5>
							<p>Proin iaculis purus consequat sem cure 
								digni ssim. Donec porttitora entum suscipit 
								aenean rhoncus posuere odio in tincidunt. Proin 
								iaculis purus consequat sem cure digni 
								ssim. Donec porttitora entum suscipit.
							</p>
						</div>
						<div class="about-img right">
							<img src="img/Image3.png" />
						</div>
					</div>
					
					<a href="#" class="about-link">Our<br />Story<br />Continues<br>...</a>
					

					
				</div>
			</div>
		</div>
		<div id="team" class="box">
			<div class="container">
				<h2 class="title">our amazing team</h2>
				<p class="descr">Proin iaculis purus consequat sem cure.</p>
				<div class="team-wrap clearfix">
					<div class="team-item">
						<img src="img/team.png" width="220" height="220" />
						<h5 class="title">Kimberly Thompson</h5>
						<p class="cat">Marketer</p>
						<div class="social">
							<a href="#" class="ic-tw">&nbsp;</a>
							<a href="#" class="ic-fb">&nbsp;</a>
							<a href="#" class="ic-pn">&nbsp;</a>
						</div>
					</div>

					<div class="team-item">
						<img src="img/team.png" width="220" height="220" />
						<h5 class="title">Kimberly Thompson</h5>
						<p class="cat">Marketer</p>
						<div class="social">
							<a href="#" class="ic-tw">&nbsp;</a>
							<a href="#" class="ic-fb">&nbsp;</a>
							<a href="#" class="ic-gl">&nbsp;</a>
						</div>
					</div>

					<div class="team-item">
						<img src="img/team.png" width="220" height="220" />
						<h5 class="title">Kimberly Thompson</h5>
						<p class="cat">Marketer</p>
						<div class="social">
							<a href="#" class="ic-fb">&nbsp;</a>
							<a href="#" class="ic-pn">&nbsp;</a>
						</div>
					</div>
				</div>
				<p class="descr team-info">Proin iaculis purus consequat sem cure  digni ssim donec porttitora entum suscipit<br />aenean rhoncus posuere odio in tincidunt proin iaculis.
				</p>
			</div>
		</div>
		<div id="brand">
			<div class="container">
				<a href="#"><img src="img/envato.jpg" width="161" height="30"></a>
				<a href="#"><img src="img/wordpress.png" width="186" height="42"></a>
				<a href="#"><img src="img/tuts.jpg" width="131" height="41"></a>
				<a href="#"><img src="img/microlancer%20copy.jpg" width="167" height="42"></a>
			</div>
		</div>
		<div id="contact" class="box">
			<div class="container">
				<h2 class="title">Contact us</h2>
				<p class="descr">Proin iaculis purus consequat sem cure.</p>
				<form action="#" id="messege">
					<div class="clearfix">
						<div class="left">
							<input type="text" placeholder="your Name *" />
							<input type="text" placeholder="your e-mail *" />
							<input type="text" placeholder="subject" />
						</div>
						<div class="right">
							<textarea placeholder="Your message *" cols="0" rows="0"></textarea>
						</div>
					</div>
					<input type="submit" value="Send Message" class="btn">
				</form>
			</div>
		</div>
		<div id="copyright">
			<div class="container clearfix">
				<p class="left">&copy; Copyright 2014 FreebiesXpress.com</p>
				<div class="social">
							<a href="#" class="ic-tw black">&nbsp;</a>
							<a href="#" class="ic-fb black">&nbsp;</a>
							<a href="#" class="ic-pn black">&nbsp;</a>
							<a href="#" class="ic-gl black">&nbsp;</a>
				</div>
			</div>
		</div>
	</div>
</body>
</html>